src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -0,0 +1,275 @@
+    end
+end
+
+_reindexset(M::Set{Int}, ind::Vector{Int}) = [ ind[x+1] for x in M ]
+
+function _characteristicvector(M::Vector{Int}, n::Int)
+    chi = zeros(Int, n)
+    chi[M] .= 1
+    return chi
+end
+
+################################################################################
+##  Properties
+################################################################################
+
+@doc Markdown.doc"""
+    nvertices(K::SimplicialComplex)
+
+Number of vertices of the abstract simplicial complex `K`.
+"""
+nvertices(K::SimplicialComplex) = pm_object(K).N_VERTICES
+
+@doc Markdown.doc"""
+    facets(K::SimplicialComplex)
+
+Maximal (by inclusion) faces of the abstract simplicial complex `K`.
+"""
+function facets(K::SimplicialComplex)
+    bigobject = pm_object(K)
+    the_facets = Vector{Set{Int}}(bigobject.FACETS)
+    return [ [x+1 for x in sigma] for sigma in the_facets ] # shift polymake indices by one
+end
+
+@doc Markdown.doc"""
+    dim(K::SimplicialComplex)
+
+Dimension of the abstract simplicial complex `K`.
+"""
+dim(K::SimplicialComplex) = pm_object(K).DIM
+
+@doc Markdown.doc"""
+    f_vector(K::SimplicialComplex)
+
+Face vector (number of faces per dimension) of the abstract simplicial complex `K`.
+"""
+f_vector(K::SimplicialComplex) = Vector{Int}(pm_object(K).F_VECTOR)
+
+@doc Markdown.doc"""
+    f_vector(K::SimplicialComplex)
+
+H-vector of the abstract simplicial complex `K`.
+"""
+h_vector(K::SimplicialComplex) = Vector{Int}(pm_object(K).H_VECTOR)
+
+@doc Markdown.doc"""
+    betti_numbers(K::SimplicialComplex)
+
+Rational Betti numbers of the abstract simplicial complex `K`.
+"""
+betti_numbers(K::SimplicialComplex) = Polymake.topaz.betti_numbers(pm_object(K))
+
+@doc Markdown.doc"""
+    euler_characteristic(K::SimplicialComplex)
+
+Reduced Euler characteristic of the abstract simplicial complex `K`.
+"""
+euler_characteristic(K::SimplicialComplex) = pm_object(K).EULER_CHARACTERISTIC
+
+@doc Markdown.doc"""
+    minimalnonfaces(K::SimplicialComplex)
+
+Minimal non-faces of the abstract simplicial complex `K`.
+
+# Example
+```jldoctest
+julia> K = SimplicialComplex([[1,2,3],[2,3,4]]);
+
+julia> minimalnonfaces(K)
+1-element Vector{Vector{Int64}}:
+ [1, 4]
+```
+"""
+function minimalnonfaces(K::SimplicialComplex)
+    bigobject = pm_object(K)
+    mnf = Vector{Set{Int}}(bigobject.MINIMAL_NON_FACES)
+    return [ [x+1 for x in f] for f in mnf ] # shift polymake indices by one
+end
+
+@doc Markdown.doc"""
+    stanley_reisner_ideal(K::SimplicialComplex)
+
+Stanley-Reisner ideal of the abstract simplicial complex `K`.
+
+# Example
+```jldoctest
+julia> stanley_reisner_ideal(realprojectiveplane())
+ideal(x1*x2*x3, x1*x2*x4, x1*x5*x6, x2*x5*x6, x1*x3*x6, x1*x4*x5, x3*x4*x5, x3*x4*x6, x2*x3*x5, x2*x4*x6)
+```
+"""
+function stanley_reisner_ideal(K::SimplicialComplex)
+    n = nvertices(K)
+    R, () = PolynomialRing(ZZ, n)
+    return ideal([ R([ZZ(1)], [_characteristicvector(f,n)]) for f in minimalnonfaces(K) ])
+end
```

Comment:
Would it make sense to also have a variant that takes the ring as an additional argument, like for `ehrhart_polynomial` and `h_star_polynomial`?